src/components/ArticlesContainer.jsx

```diff
@@ -42,19 +45,26 @@ export default function ArticlesContainer() {
           return err.message;
         });
       });
-    getPageNumbers().then((pageBtnValues) => {
+    getPageNumbers(undefined, topicName).then((pageBtnValues) => {
       setPages(() => {
         return pageBtnValues;
       });
     });
-  }, [pageNum]);
+  }, [pageNum, topicName]);
 
   return (
     <>
-      <h1>Articles</h1>
+      {isTopic ? (
```

Comment:
Good way to handle the two different types of parameters you can have on this endpoint